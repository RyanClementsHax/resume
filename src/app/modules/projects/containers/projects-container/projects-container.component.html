<p>
  projects-container works!
</p>
