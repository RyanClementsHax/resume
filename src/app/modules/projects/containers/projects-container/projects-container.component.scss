@import "variables.scss";

:host {
    display: block;
    padding: $content-padding;

    h1 {
        text-align: center;
    }
    
    .project-list {
        display: flex;
        flex-direction: column;
        align-items: center;

        article {
            margin-bottom: 50px;
        }
    }
}