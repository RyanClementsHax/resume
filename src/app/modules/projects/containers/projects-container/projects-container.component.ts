import { Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-projects-container',
  templateUrl: './projects-container.component.html',
  styleUrls: ['./projects-container.component.scss']
})
export class ProjectsContainerComponent implements OnInit {

  constructor() { }

  ngOnInit() {
  }

}
