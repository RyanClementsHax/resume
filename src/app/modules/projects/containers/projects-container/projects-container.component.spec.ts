import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { ProjectsContainerComponent } from './projects-container.component';

describe('ProjectsContainerComponent', () => {
  let component: ProjectsContainerComponent;
  let fixture: ComponentFixture<ProjectsContainerComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [ ProjectsContainerComponent ]
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(ProjectsContainerComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });
});
