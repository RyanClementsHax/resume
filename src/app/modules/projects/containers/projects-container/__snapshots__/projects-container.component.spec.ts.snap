// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectsContainerComponent should match snapshot 1`] = `
ProjectsContainerComponent {
  "projects$": Subject {
    "_isScalar": false,
    "closed": false,
    "hasError": false,
    "isStopped": false,
    "observers": Array [
      Subscriber {
        "_parentOrParents": null,
        "_subscriptions": Array [
          SubjectSubscription {
            "_parentOrParents": [Circular],
            "_subscriptions": null,
            "closed": false,
            "subject": [Circular],
            "subscriber": [Circular],
          },
        ],
        "closed": false,
        "destination": SafeSubscriber {
          "_complete": undefined,
          "_context": Object {
            "unsubscribe": [Function],
          },
          "_error": [Function],
          "_next": [Function],
          "_parentOrParents": null,
          "_parentSubscriber": [Circular],
          "_subscriptions": null,
          "closed": false,
          "destination": Object {
            "closed": true,
            "complete": [Function],
            "error": [Function],
            "next": [Function],
          },
          "isStopped": false,
          "syncErrorThrowable": false,
          "syncErrorThrown": false,
          "syncErrorValue": null,
        },
        "isStopped": false,
        "syncErrorThrowable": true,
        "syncErrorThrown": false,
        "syncErrorValue": null,
      },
    ],
    "thrownError": null,
  },
  "projectsService": MockProjectsService {
    "projects$": Subject {
      "_isScalar": false,
      "closed": false,
      "hasError": false,
      "isStopped": false,
      "observers": Array [
        Subscriber {
          "_parentOrParents": null,
          "_subscriptions": Array [
            SubjectSubscription {
              "_parentOrParents": [Circular],
              "_subscriptions": null,
              "closed": false,
              "subject": [Circular],
              "subscriber": [Circular],
            },
          ],
          "closed": false,
          "destination": SafeSubscriber {
            "_complete": undefined,
            "_context": Object {
              "unsubscribe": [Function],
            },
            "_error": [Function],
            "_next": [Function],
            "_parentOrParents": null,
            "_parentSubscriber": [Circular],
            "_subscriptions": null,
            "closed": false,
            "destination": Object {
              "closed": true,
              "complete": [Function],
              "error": [Function],
              "next": [Function],
            },
            "isStopped": false,
            "syncErrorThrowable": false,
            "syncErrorThrown": false,
            "syncErrorValue": null,
          },
          "isStopped": false,
          "syncErrorThrowable": true,
          "syncErrorThrown": false,
          "syncErrorValue": null,
        },
      ],
      "thrownError": null,
    },
  },
}
`;
